scraper_common.js

```diff
@@ -75,7 +75,16 @@ async function execute(usePuppeteer, scrapers) {
                 startTime,
                 numberAppointments
             );
+
+            // Add the current timestamp to each individual location
+            const nowTimestamp = moment().format();
+            returnValue?.individualLocationData.map((loc) => {
+                loc.timestamp = nowTimestamp;
```

Comment:
I realize we now no longer have anybody setting the timestamp, but there's an argument that says that if there is already a timestamp from the scraper, that the scraper knows better and we ought not override it.

However, that attitude lead to past problems with CVS, where we presented CVS's timestamp and the result was the frontend concluded out data was stale, when it wasn't, or…at least, it was as fresh as it could be.

So perhaps there should be a diagnostic emitted if there is an existing timestamp key.
Maybe that's too much, I don't know.

Reply:
All of the scrapers had removed the timestamp, and even then it was just `moment().format()`.  CVS appears to be the only one and we are passing that as `siteTimestamp` (although we don't use it for anything right now)